connect/src/com/telenor/mobileconnect/MobileConnectSdkProfile.java

```diff
@@ -184,17 +133,17 @@ private void tryMccMnc(
                 operatorDiscoveryConfig.getOperatorDiscoveryRedirectUri(),
                 mcc,
                 mnc,
-                odCallbackForMccMnc);
-    }
+                new Callback<OperatorDiscoveryAPI.OperatorDiscoveryResult>() {
```

Comment:
Great that u did that with the anonymous callbacks :)
I wanted to write that a few times and I always was interrupted by something.